src/main/kotlin/com/exactpro/th2/kafka/client/Config.kt

```diff
@@ -104,4 +127,7 @@ class Config(
     }
 }
 
-data class KafkaStream(val topic: String, val key: String?)
\ No newline at end of file
+data class KafkaStream(
+    @JsonProperty(required = true) val topic: String,
+    @JsonProperty(required = true) val key: String?
```

Comment:
Why is it required?

Reply:
To avoid misunderstanding: if no key specified in config file it may look like that key is ignored and all messages from the topic match the criteria (and this is wrong), so i request to specify null key - and it will look like that only messages with null key match the criteria (and this is how it actually works).